CHANGELOG.md

```diff
@@ -10,14 +10,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Kusama: Relay General Admin Origin mapping to xcm Location ([polkadot-fellows/runtimes#383](https://github.com/polkadot-fellows/runtimes/pull/383))
 - Encointer, PeopleKusama, PeoplePolkadot: Configure delivery fees for UMP ([polkadot-fellows/runtimes#390](https://github.com/polkadot-fellows/runtimes/pull/390))
-
-### Changed
-
-- Bounties: Remove payout delay ([polkadot-fellows/runtimes#386](https://github.com/polkadot-fellows/runtimes/pull/386))
-
-## [1.2.8] 03.07.2024
```

Comment:
I think the changelog on master is not correct, since it shows that 1.2.8 would have released a lot, which it AFAIK did not: https://github.com/polkadot-fellows/runtimes/blob/main/CHANGELOG.md  
Introduce in: https://github.com/polkadot-fellows/runtimes/commit/2c13e088f0c2c1d81942d487055e67e1162a45ba

Reply:
I see, directly destroyed my pr before: https://github.com/polkadot-fellows/runtimes/commit/ce7838d2263a66f47718a373eceef2baf9e53257